internal/services/azapi_resource.go

```diff
@@ -196,6 +197,12 @@ func (r *AzapiResource) Schema(ctx context.Context, _ resource.SchemaRequest, re
 				},
 			},
 
+			"replace_triggers_refs": schema.ListAttribute{
```

Comment:
Please also implement this feature in the data plane resource.

Reply:
Got it, any need for other resources too?

Reply:
Thanks! No.

Reply:
Added & tested